therapy/etl/hemonc.py

```diff
@@ -175,6 +175,8 @@ def _get_rels(self, therapies: Dict, brand_names: Dict,
                 label = conditions[row[1]]
                 norm_response = self.disease_normalizer.search_groups(label)
                 if norm_response['match_type'] > 0:
+                    # TODO: Change once disease normalizer updated
+                    # ncit_id = norm_response['disease_descriptor']['disease_id']  # noqa: E501
```

Comment:
Let's just push the disease norm update through to PyPI and then we can update this/pin the new version in the Pipfile

Reply:
Ya I was going to wait until disease got merged!